### PR TITLE
Add poppler-data

### PR DESCRIPTION
**Pull request by superloach**
Working toward #2580

### CODE REVIEW

File: packages/poppler_data.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  description 'The additional package consists of encoding files for use with Poppler.'
+  homepage 'https://poppler.freedesktop.org/'
+  version '0.4.9'
+  source_url 'https://poppler.freedesktop.org/poppler-data-0.4.9.tar.gz'
+  source_sha256 '1f9c7e7de9ecd0db6ab287349e31bf815ca108a5a175cf906a90163bdbe32012'
+
+  depends_on 'poppler'
+
+  def self.install
+    system "make prefix=#{CREW_DEST_DIR} install"
```
[the remark this answers is not in the document]
Comment (uberhacker) [reply packages/poppler_data.rb]: In general, however, you should have a `make` in `def self.build` and use `DESTDIR=#{CREW_DEST_DIR}` in `def self.install`.  Some source code doesn't enable you to configure in an orthodox way, so I completely understand if this is not an option sometimes.

**Comment by uberhacker** [comment]
@superloach: I just extracted the source tarball and it looks like you need to install via cmake.  The freeglut package has an example of how you can install using cmake.  There are several other packages you can examine that use cmake also if it helps any.

**Comment by superloach** [comment]
Alright, I'll work on that when I get home. Thanks again for all your help in figuring these things out, I'm so glad I can help now :)

**Comment by uberhacker** [comment]
No problem and we appreciate all your efforts!

**Comment by superloach** [comment]
@uberhacker I see what you mean about cmake, but the README states:
> "All the Makefile does is install the data files, so autotools seemed a bit overkill.  There is no ./configure file, all there is to do is to say 'make install'.  To override the install destinations, specify prefix, datadir, pkgdatadir or DESTDIR at 'make install' time as desired."

That's why I only used the `make install` command.
However, I did just notice that I should've used `--prefix` instead of `prefix`.

**Comment by uberhacker** [comment]
Fair enough.  Glad you are reading those INSTALL/README files. :)

**Pull request merged**